bmarlow/Week 4/dict_lab.py

```diff
@@ -0,0 +1,110 @@
+# fun with dictionaries
+def main():
+    dictionary = {
+        'Name': 'Chris',
+        'City': 'Seattle',
+        'Cake': 'Chocolate',
+    }
+    print('Printing the full dictionary')
+    print(dictionary)
+    print('')
+
+    print('Removing the Cake')
+    dictionary.pop('Cake')
+    print(dictionary)
+    print('')
+
+    print('Adding the fruit')
+    dictionary['Fruit'] = 'Mango'
+    print(dictionary)
+    print('')
+
+    print('Showing the keys')
+    for key in dictionary:
+        print(key)
+    print('')
+
+
+    print('Showing the values')
+    for value in dictionary.values():
+        print(value)
+    print('')
+
+    print('Is Cake still a valid key?')
+    cakeexist = 'Cake' in dictionary
+    print(cakeexist)
+    print('')
+
+    print('Is Mango still a valid value?')
+    mangoexist = 'Mango' in dictionary.values()
+    print(mangoexist)
+    print('')
+
+    print('How often is the letter t used in each value?')
+    tempdict = {}
+    newdict = {}
+    dictionary2 = {
+        'Name': 'Chris',
+        'City': 'Seattle',
+        'Cake': 'Chocolate',
+    }
+
+    for key in dictionary2:
+        count = dictionary2[key].count('t')
+        tempdict = {key: count}
+        newdict.update(tempdict)
```

Comment:
you can take the above two lines and do it in one via `newdict.update({key:count})`